tests/Integration/UI/AdminBarTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Admin bar tests.
+ *
+ * @package Parsely
+ */
+
+declare(strict_types=1);
+
+namespace Parsely\Tests\Integration\UI;
+
+// Explicit require as the `Rest` class could not be found in some environments.
+require_once __DIR__ . '/../../../src/UI/class-admin-bar.php';
```

Comment:
Is there any more background on why we have to explicitly require UI classes such as this in an issue / PR somewhere? Just curious.

Reply:
I have removed this line as it works fine in CI. This should be tackled in #586